packages/devextreme/js/ui/html_editor/modules/mentions.js

```diff
@@ -313,6 +313,10 @@ if(Quill) {
                 itemTemplate,
                 searchExpr
             });
+
+            const $container = this._list._$container;
+            this._list.option('focusedElement', getPublicElement($container));
```

Comment:
why do you set focusedElement here?

Reply:
That triggers the dx-scrollview-content to generate the ID

Reply:
this is not a way to generate id ) anyway, look at my other comment